customize.dist/translations/messages.pl.js

```diff
@@ -0,0 +1,229 @@
+                    '<p>Uważaj by nie zapomnieć swoich danych, odzyskanie ich jest niemożliwe.</p>';
+
+    out.forget = "Zapomnij";
+
+    // Polls
+
+    out.poll_title = "Prosty koordynator planu dnia"; // Choice of "Koordynator" can be discussed
+    out.poll_subtitle = "Proste planowanie tygodnia, <em>w czasie rzeczywistym</em>";
+
+    out.poll_p_save = "Twoje ustawienia aktualizowane są na bierząco. Nie martw się o zapisywanie.";
+    out.poll_p_encryption = "Wszystko co robisz jest szyfrowane, więc tylko osoby z linkiem mają tu dostęp. Nawet serwer nie widzi co kombinujesz.";
+    out.poll_p_howtouse = "Wpisz swoje imię w polu poniżej i zaznacz kiedy będziesz dostępny.";
+
+    out.promptName = "Jak się nazywasz ?";
+
+    out.wizardButton = 'KREATOR';
+    out.wizardLog = "Naciśnij przycisk w lewym-górnym rogu by wrócić do planu";
+    out.wizardTitle = "Uzyj kreatora by stworzyć opcje do głosowanie";
+    out.wizardConfirm = "Jesteś pewny, że chcesz dodać te opcje do głosowania?";
+
+    out.poll_closeWizardButton = "Zamknij kreator";
+    out.poll_closeWizardButtonTitle = "Zamyka kreator";
+    out.poll_wizardComputeButton = "Ustawienia kalkulacji";
+    out.poll_wizardClearButton = "Wyczyść tabelę";
+    out.poll_wizardDescription = "Automatycznie stwórz część opcji poprzez wpisanie ilości dat i godzin";
+    out.poll_wizardAddDateButton = "+ Daty";
+    out.poll_wizardAddTimeButton = "+ Godziny";
+
+    out.poll_addUserButton = "+ Użytkownik";
+    out.poll_addUserButtonTitle = "Naciśnij, by dodać użytkownika";
+    out.poll_addOptionButton = "+ Opcja";
+    out.poll_addOptionButtonTitle = "Naciśnij, by dodać opcję";
+    out.poll_addOption = "Zaproponuj opcję";
+    out.poll_optionPlaceholder = "Opcja";
+    out.poll_addUser = "Podaj imię";
+    out.poll_userPlaceholder = "Twoje imię";
+    out.poll_removeOption = "Jesteś pewien, że chcesz usunąć tę opcję?";
+    out.poll_removeOptionTitle = "Usuń wiersz";
+    out.poll_removeUser = "Jesteś pewien, że chcesz usunąć tego użytkownika?";
+    out.poll_removeUserTitle = "Usuń kolumnę";
+    out.poll_editOption = "Jesteś pewien, że chcesz edytować tę opcję?";
+    out.poll_editOptionTitle = "Edytuj wiersz";
+    out.poll_editUser = "Jesteś pewien, że chcesz edytować tego uzytkownika?";
+    out.poll_editUserTitle = "Edytuj kolumnę";
+
+    out.poll_titleHint = "Tytuł";
+    out.poll_descriptionHint = "Opis";
+
+    // index.html
+
+    out.main_p1 = 'CryptPad to edytor czasu rzeczywistego stawiający na wspólpracę i <strong>prostotę użytkowania</strong>. Szyfrowanie w twojej przeglądarce, chroni twoje dane przed serwerem, chmurą i ABW. Tajny klucz jest przechowywany w adresie strony <a href="https://pl.wikipedia.org/wiki/Uniform_Resource_Identifier#Pe.C5.82ny_wz.C3.B3r_URI_hierarchicznych.5B2.5D">identyfikator fragmentu w URI</a> którego nigdy nie przekazujemy serwerowi, a dostępny jest dla javascript. Udostępniając URL(link), przekazujesz innym upoważnienie do uczestnictwa.';
```

Comment:
NSA has nothing to do in Poland. It's not their jurisdiction and ABW is their equivalent in Poland.
That's a good idea with square brackets and URI fragment. I'd add it.